cmd/carbonapi/http/render_handler.go

```diff
@@ -309,7 +314,7 @@ func renderHandler(w http.ResponseWriter, r *http.Request) {
 		if returnCode == 404 {
 			returnCode = config.Config.NotFoundStatusCode
 		}
-		if returnCode >= 500 {
+		if returnCode >= 500 || returnCode == 400 {
```

Comment:
I actually wonder if it won't be better to have a separate log message for error code 400